008_team_6_agent_design/008_team_6_agent_design.tex

```diff
@@ -1 +1,1021 @@
-\chapter{Team 6 Agent Design}\label{team_6_agent_design}
\ No newline at end of file
+\chapter{Team 6 Agent Design}\label{team_6_agent_design}
+
+\section{Overview}
+
+In this chapter, we present the technical formulation and results of the Team 6 agent. This chapter is divided into 7 parts and is structured as follows: 
+\begin{enumerate}
+    \item Specification of the agent structure (\Cref{agent-specification}) .
+    \item Introduction of the underlying concepts upon which the agent's behaviour is based (\Cref{social_motives}). 
+    \item Demonstration of how the agents utilise common pool resources according to the concepts defined in the previous section (\Cref{food_consumption}). 
+    \item Explanation of how a social network is formed through the concept of trust (\Cref{trust}).
+    \item Exploration of how the agents communicate and interact with each other in order to self-organize (\Cref{self-organization}).
+    \item Presentation and discussion of a set of experimental results through simulations (\Cref{simulation_results}).
+    \item Conclusion and an outlook to future work (\Cref{conclusion_future_work}).
+\end{enumerate}
+
+Ultimately, we propose that this agent accurately parallels aspects of human nature through its capability to undergo behavioural change and formulate social connections with trust, as well as heavily and intelligently utilizes treaties with an outlook to self-organization.
+
+%%%%%%%%%
+%%%%%%%%% AGENT SPECIFICATION
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+\section{Specification of Agent Design}\label{agent-specification}
+
+All agents \textit{i} $\in$ \textit{A} are implemented as a data structure, such that all agents contain sufficient parameterisation to participate in the various communication methods \textit{c} $\in$ \textit{C}, resulting in a set of interactions defined by I = $< \mathit{A, C} >$. Each agent inherits from the \textit{baseAgent} structure, before inheriting the fields contained in table \ref{tab:agentStruct}. We note only the most relevant fields for quantifying the agent have been included.
```

Comment:
You don't seem to define what `i` and `A` are here?

`baseAgent` is code and should use `\texttt` imo

Reply:
baseAgend texttt changed